src/lib/GovernorTimelockCompound.sol

```diff
@@ -0,0 +1,256 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.6.0) (governance/extensions/GovernorTimelockCompound.sol)
+//
+// The Pooltogether Timelock is not compatible with the Governor Alpha Timelock interface. This
+// meant we had to fork the contracts where the timelock is needed and replace the Timelock
```

Comment:
nit: "meant we had to fork the contracts *that interact with the timelock* and replace..."

Reply:
https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/13/commits/f550efb50c719277d68909a630e47ca812409e43